types/node/index.d.ts

```diff
@@ -1439,13 +1439,27 @@ declare module "zlib" {
         dictionary?: any; // deflate/inflate only, empty dictionary by default
     }
 
-    export interface Gzip extends stream.Transform { }
-    export interface Gunzip extends stream.Transform { }
-    export interface Deflate extends stream.Transform { }
-    export interface Inflate extends stream.Transform { }
-    export interface DeflateRaw extends stream.Transform { }
-    export interface InflateRaw extends stream.Transform { }
-    export interface Unzip extends stream.Transform { }
+    export interface Zlib {
+        readonly bytesRead: number;
+        close(callback?: Function): void;
```

Comment:
I think callback is of type `() => void`.

Reply:
It doesn't say in the documentation. When I try, I get an empty object as argument.
Since it's very unclear by node's documentation, wouldn't `Function` be preferred?